content/300-guides/400-migrate-to-prisma/02-migrate-from-sequelize.mdx

```diff
@@ -403,7 +403,7 @@ model Profiles {
   bio       String
   createdAt DateTime
   updatedAt DateTime
-  userId    Int?
+  userId    Int?     @unique
```

Comment:
Are we sure the SQL here is actually introspected with `@unique` and is not invalid? That might require some mention that this needs to be fixed here then.

Reply:
Yeah, since we're starting from a Sequelize project and I can't see the SQL being introspected I'm also not sure here. Would need to run the actual example.

Reply:
Leaving this as is for now